code/modules/clothing/under/pants.dm

```diff
@@ -46,7 +46,7 @@
 	desc = "Some tan pants. You look like a white collar worker with these on."
 	icon_state = "tanpants"
 
-/obj/item/clothing/under/pants/polypants/polychromic
+/obj/item/clothing/under/pants/polypants
```

Comment:
remove the polychromic subtype on the below ComponentInitialize() as well or else it won't have the polychromic element